src/main/java/com/palantir/docker/compose/execution/AggressiveShutdownStrategy.java

```diff
@@ -27,25 +27,15 @@ public void shutdown(DockerComposeRule rule) throws IOException, InterruptedExce
         List<ContainerName> runningContainers = rule.dockerCompose().ps();
 
         log.info("Shutting down {}", runningContainers.stream().map(ContainerName::semanticName).collect(toList()));
-        if (removeContainersCatchingErrors(rule, runningContainers)) {
-            return;
-        }
-
-        log.debug("First shutdown attempted failed due to btrfs volume error... retrying");
-        if (removeContainersCatchingErrors(rule, runningContainers)) {
-            return;
-        }
-
-        log.warn("Couldn't shut down containers due to btrfs volume error, "
-                + "see https://circleci.com/docs/docker-btrfs-error/ for more info.");
+        removeContainersCatchingErrors(rule, runningContainers);
```

Comment:
I don't think we can get rid of the retry logic in Aggressive.  The BTRFS retries were added in https://github.com/palantir/docker-compose-rule/pull/115 because Circle occasionally fails to shut down containers.  If we don't retry, then leftover containers will conflict with later tests.

Could we either: 
- Keep the retries in ShutdownStrategy.AGGRESSIVE
- Add the network behaviour to a new ShutdownStrategy (leaving the current aggressive unchanged)
- Or, use your custom ShutdownStrategy on a few circle builds (before merging into DCR) and convince us that the BTRFS retries aren't needed